core/exec.c

```diff
@@ -1111,7 +1112,7 @@ static lai_api_error_t lai_exec_process(lai_state_t *state) {
             int initial_size = block->limit - block->pc;
             if (initial_size < 0)
                 lai_panic("buffer initializer has negative size");
-            if (initial_size > lai_exec_buffer_size(&result))
+            if (initial_size > (int) lai_exec_buffer_size(&result))
```

Comment:
It'd make more sense to me to make `initial_size` a size_t here rather than casting lai_exec_buffer_size to an int

Reply:
`initial_size` has to be signed (see line 1113) and therefore I think it'd make more sense to use int.